### PR TITLE
Replace all sysouts

### PR DESCRIPTION
**Pull request by Warriorrrr**
#### Description: 

Replaces all uses of System.out/err with getLogger().info/warn
____
#### New Nodes/Commands/ConfigOptions: 

____
#### Relevant Towny Issue ticket:

____
- [x] I have tested this pull request for defects on a server. 

By making this pull request, I represent that I have the right to waive copyright and related rights to my contribution, and agree that all copyright and related rights in my contributions are waived, and I acknowledge that the TownyAdvanced organization has the copyright to use and modify my contribution under the Towny [License](https://github.com/LlmDl/Towny/blob/master/LICENSE.md) for perpetuity.

### CODE REVIEW

**Review by TheFlagCourier — COMMENTED** [review]
Would it be more prudent to set the logger as a constant?

File: src/com/palmergames/bukkit/towny/Towny.java [hunk 1]
```diff
@@ -469,19 +466,19 @@ private void printChangelogToConsole() {
...
+			plugin.getLogger().info("ChangeLog up until v" + getVersion());
 			String lastVersion = Version.fromString(TownySettings.getLastRunVersion()).toString(); // Parse out any trailing text after the *.*.*.* version, ie "-for-1.12.2".
 			for (String line : changeLog) { // TODO: crawl from the bottom, then
 											// past from that index.
 				if (line.startsWith(lastVersion)) {
 					display = true;
 				}
 				if (display && line.replaceAll(" ", "").replaceAll("\t", "").length() > 0) {
-					System.out.println(line);
+					plugin.getLogger().info(line);
```
Comment (LlmDl) [inline src/com/palmergames/bukkit/towny/Towny.java:478]: This one should be Bukkit.getLogger()

File: src/com/palmergames/bukkit/towny/db/TownyDatabaseHandler.java [hunk 2]
```diff
@@ -139,10 +139,10 @@ public void finishTasks() {
...
 
 		if (!TownySettings.getSaveDatabase().equalsIgnoreCase("flatfile")) {
-			System.out.println("***** Warning *****");
-			System.out.println("***** Only Snapshots & Regen files in towny\\data\\ will be backed up!");
-			System.out.println("***** This does not include your residents/towns/nations.");
-			System.out.println("***** Make sure you have scheduled a backup in MySQL too!!!");
+			plugin.getLogger().warning("***** Warning *****");
+			plugin.getLogger().warning("***** Only Snapshots & Regen files in towny\\data\\ will be backed up!");
+			plugin.getLogger().warning("***** This does not include your residents/towns/nations.");
+			plugin.getLogger().warning("***** Make sure you have scheduled a backup in MySQL too!!!");
```
Comment (LlmDl) [inline src/com/palmergames/bukkit/towny/db/TownyDatabaseHandler.java:145]: Consider making this not actually warn, remain in white. We don't want to train people to ignore red/yellow during the towny startup.

**Review by LlmDl — CHANGES_REQUESTED** [review]

**Comment by LlmDl** [comment]
> Would it be more prudent to set the logger as a constant?

I could go either way on this. We already have a very under-used log(String) method in the Towny class. We could add warn(String) and severe(String) to get the other logging options.

**Pull request merged**